internal/test/scorecard/tests.go

```diff
@@ -49,14 +49,16 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes/scheme"
 )
 
 const (
 	OperatorInstallTestName string        = "operator-install"
 	CryostatCRTestName      string        = "cryostat-cr"
 	operatorDeploymentName  string        = "cryostat-operator-controller-manager"
-	testTimeout             time.Duration = time.Minute * 10
+	testTimeout             time.Duration = time.Minute * 2 // FIXME
```

Comment:
FIXME?

Reply:
Oops! Reverted this back. Thanks for noticing